test/cypress/integration/29-host.admin.test.js

```diff
@@ -157,6 +157,7 @@ describe('host dashboard', () => {
       // Mark as expired
       cy.getByDataCy('MARK_AS_EXPIRED-button').click();
       cy.get('[data-cy="MARK_AS_EXPIRED-confirmation-modal"] [data-cy="confirmation-modal-continue"]').click();
+      cy.wait(500);
```

Comment:
I still haven't nailed properly while this test is failing, but a `wait` won't help here - the page is blocked with a wrong status and won't update no matter how much time we wait. I was working on this with https://github.com/opencollective/opencollective-frontend/pull/9531, it should improve the UX and help us understand why it's failing.